backend/src/main/java/gov/cdc/usds/simplereport/service/DeviceTypeService.java

```diff
@@ -80,19 +95,26 @@ public DeviceType createDeviceType(
         String loincCode,
```

Comment:
are we okay with the prospect of facilities having to go through us to register new device types? i suppose this beats the alternative of having a big mess of facilities trying to register similar device types on their own.

Reply:
Allowed device (and specimen) types are actually managed by CDC—we get the data from https://www.cdc.gov/csels/dls/sars-cov-2-livd-codes.html. In a world where we had excess development resources, we might try parsing the spreadsheet for device types directly, but in no event do the users get to make up their own devices. 😁